tests/security/cc/trustedprograms.pm

```diff
@@ -42,7 +42,7 @@ sub run {
 
     select_console 'root-console';
 
-    run_testcase('trustedprograms', (make => 1, timeout => 900));
+    run_testcase('trustedprograms', (make => 1, timeout => 960));
```

Comment:
@Amrysliu 
I would suggest add more timeout e.g. 1200 for aarch64 platform, and set timeout=900 for others

Reply:
According to https://openqa.suse.de/tests/7882864#step/trustedprograms/38, the command almost completely run. So I think increasing some seconds should work. Different timeout in different platforms make it complex. So I prefer to keep that simply.

Reply:
I agree with you on this part, however, it is an known issue that we have lower performance on arm workers, increase only 1m can't make sure we can cover "more", I mean we should use more buffer here :)